src/coreclr/jit/compiler.cpp

```diff
@@ -4636,7 +4636,7 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
         // Run an early flow graph simplification pass
         //
         auto earlyUpdateFlowGraphPhase = [this]() {
-            const bool doTailDup = false;
+            constexpr bool doTailDup = false;
```

Comment:
Just curious, what is the rationale behind changing const to constexpr?

Reply:
I guess just to ensure it's a compile-time constant expression. This pattern is really only used to have named boolean arguments. Another option is `foo(/* doTailDup */ false)` style calls with a comment that shows the parameter name.